.github/workflows/nrn-modeldb-ci.yaml

```diff
@@ -28,19 +51,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      matrix:
-        python-version: ["3.9"]
     env:
       DISPLAY: ${{ ':0' }}
-      NEURON_V1: ${{ github.event.inputs.neuron_v1 || 'neuron' }}
-      NEURON_V2: ${{ github.event.inputs.neuron_v2 || 'neuron-nightly' }}
-      MODELS_TO_RUN: ${{ github.event.inputs.models_to_run || '' }}
+      NEURON_V1: ${{ github.event.inputs.neuron_v1 || inputs.neuron_v1 || 'neuron' }}
+      NEURON_V2: ${{ github.event.inputs.neuron_v2 || inputs.neuron_v2 || 'neuron-nightly' }}
+      MODELS_TO_RUN: ${{ github.event.inputs.models_to_run || inputs.neuron_v2 || '' }}
```

Comment:
This should probably be `inputs.models_to_run`

Reply:
good catch, thanks